packages/flutter_tools/lib/src/device.dart

```diff
@@ -410,17 +410,30 @@ abstract class Device {
   /// Whether the device is supported for the current project directory.
   bool isSupportedForProject(FlutterProject flutterProject);
 
-  /// Check if a version of the given app is already installed
-  Future<bool> isAppInstalled(covariant ApplicationPackage app);
+  /// Check if a version of the given app is already installed.
+  /// Specify [userIdentifier] to check if installed for a particular user (Android only).
```

Comment:
dartdoc should have a newline between the top level comment and additional comments:

```
/// Check if a version of the given app is already installed.
///
/// Specify [userIdentifier] to check if installed for a particular user (Android only)
```